classes/PaymentModule.php

```diff
@@ -1134,7 +1135,7 @@ protected function createOrderCartRules(
                         '{voucher_num}' => $voucher->code,
                         '{firstname}' => $this->context->customer->firstname,
                         '{lastname}' => $this->context->customer->lastname,
-                        '{id_order}' => $order->reference,
+                        '{id_order}' => $order->id,
```

Comment:
This is logical. However ... aren't we going to break modules that have been using this variable 😅 ?

ping @eternoendless : this would be kind of a BC break

Reply:
It's not a BC, it's a bug fix. Otherwise all our bugs fixes are BC break :sweat_smile: